src/Components/Admins/Form/index.js

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect } from 'react';
+import styles from './form.module.css';
+
+const params = new URLSearchParams(window.location.search);
+const adminId = params.get('_id');
+let fetchMethod = 'POST';
+let url = `${process.env.REACT_APP_API}/admins`;
+
+function Form() {
+  const [nameValue, setNameValue] = useState('');
+  const [usernameValue, setUsernameValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+
+  const getAdminById = () => {
+    fetch(`${process.env.REACT_APP_API}/admins?_id=${adminId}`)
+      .then((response) => {
+        if (response.status !== 200 && response.status !== 201) {
+          return response.json().then(({ message }) => {
+            throw new Error(message);
+          });
+        }
+        return response.json();
+      })
+      .then((response) => {
+        const { name, username, password } = response.data[0];
+        setNameValue(name);
+        setUsernameValue(username);
+        setPasswordValue(password);
+      })
+      .catch((error) => {
+        console.log('err', error);
+      });
+  };
+
+  useEffect(() => {
+    if (adminId) {
+      url += `/${adminId}`;
```

Comment:
save the URL into let and then use += to complete the URL is more complex than it could resolve. Just put the URL string on each case, also is easier to understand.

Reply:
Done!!